radicle/src/storage/git.rs

```diff
@@ -413,6 +414,16 @@ impl Repository {
 
         Ok(refs)
     }
+
+    pub fn resolve_oid(&self, oid: cob::ObjectId) -> Result<cob::ObjectId, git2::Error> {
+        let mut object_id = oid.to_string();
+        if object_id.starts_with("00") {
```

Comment:
We don't need to convert to string to check this, we should be able to do `oid.as_bytes().starts_with(&[0])` I think?

Reply:
Yes, this required adding the method as_bytes inside the ObjectId but I think it is the right call, thanks